models/mailchimp/mailchimp_campaigns.sql

```diff
@@ -0,0 +1,3 @@
+select distinct
```

Comment:
there are currently no instances where a campaign has multiple records in this table, so the distinct isn't needed. are you just writing it this way to protect against some unintended problem? that seems reasonable i guess, just want to make sure i understand the reasoning, because distincts can be expensive.